index.html

```diff
@@ -5,8 +5,55 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet">
 </head>
 <body>
-    
+    <header class="header">
```

Comment:
Your html page is clean and well organized! I see that you are using span and div tags for a few things. I want to share a resource that walks through when to use particular tags:
![html5_element_flowchart](https://user-images.githubusercontent.com/60710670/175045859-4728f1fa-dc82-4f4d-831f-aad4b17acfe8.png)